src/utils/imitatefeatures.js

```diff
@@ -37,12 +43,15 @@ export function imitateFeatures( editor ) {
 
 	editor.ui.featureComponents.add( 'bold', Button, ButtonView, boldModel );
 
+	// -------------------------------------------------------------------------------------------
+
 	/* istanbul ignore next */
 	const italicModel = new Model( {
 		isEnabled: true,
 		isOn: false,
 		label: t( 'Italic' ),
-		icon: 'italic'
+		icon: 'italic',
+		iconAlign: 'left'
```

Comment:
Should be uppercase `'LEFT'`. We define elements of dictionaries using `'UPPER_CASE'` strings.